<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryGame</class>
 <widget class="QMainWindow" name="MemoryGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Montserrat</family>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>NeuroGame - Memoria</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(254, 245, 240);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>1193</width>
      <height>625</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="card_1">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>17</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_3">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>473</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_19">
     <property name="geometry">
      <rect>
       <x>252</x>
       <y>167</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_6">
     <property name="geometry">
      <rect>
       <x>252</x>
       <y>317</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_2">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>14</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_8">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>470</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_5">
     <property name="geometry">
      <rect>
       <x>492</x>
       <y>17</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_10">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>170</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_11">
     <property name="geometry">
      <rect>
       <x>488</x>
       <y>473</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_12">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>320</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_13">
     <property name="geometry">
      <rect>
       <x>718</x>
       <y>173</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_14">
     <property name="geometry">
      <rect>
       <x>718</x>
       <y>323</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_15">
     <property name="geometry">
      <rect>
       <x>716</x>
       <y>476</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_7">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>20</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_17">
     <property name="geometry">
      <rect>
       <x>948</x>
       <y>173</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_18">
     <property name="geometry">
      <rect>
       <x>948</x>
       <y>323</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_9">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>20</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_20">
     <property name="geometry">
      <rect>
       <x>946</x>
       <y>476</y>
       <width>215</width>
       <height>135</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/assets/images/pattern.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="timer_text">
    <property name="geometry">
     <rect>
      <x>923</x>
      <y>34</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 57, 132);</string>
    </property>
    <property name="text">
     <string>timer</string>
    </property>
   </widget>
   <widget class="QLabel" name="Score">
    <property name="geometry">
     <rect>
      <x>1043</x>
      <y>17</y>
      <width>67</width>
      <height>66</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/star.png);
color: rgb(221, 122, 179);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="music">
    <property name="geometry">
     <rect>
      <x>1149</x>
      <y>35</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/music.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>41</y>
      <width>581</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 57, 132);</string>
    </property>
    <property name="text">
     <string>¡Toca las cartas para encontrar los pares!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sound">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>35</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/volume_down.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
